### ISSUE TITLE
[Mobile] Develop Dynamic Accessibility Scaling Engine

### ISSUE THREAD

**Issue by yosemite01** [yosemite01/stellar-creator-portfolio#616]
### Strategic Impact
Accessibility should be first-class.

### Implementation Requirements
- [ ] Respond dynamically to global font scale percentages
- [ ] Implement semantic traversal for Screen Readers natively
- [ ] Check contrast ratios programmatically

**Comment by Nuel-ship-it** [comment]
@Nuel-ship-it has applied to work on this issue as part of the **Stellar Wave Program**'s 5th wave.



> Hi maintainer, please can you assign me this? Thank you


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/8399e509-9522-462c-99a4-18c6cf79401d) or assign @Nuel-ship-it to this issue.

**Comment by Phantomcall** [comment]
@Phantomcall has applied to work on this issue as part of the **Stellar Wave Program**'s 5th wave.



> i would love to work on this


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/8399e509-9522-462c-99a4-18c6cf79401d) or assign @Phantomcall to this issue.
